### PR TITLE
Adapt to MG syntax changes

### PR DESCRIPTION
**Pull request by pstutz**

### CODE REVIEW

**Comment by pstutz** [comment]
Reqiures a release of https://github.com/opencypher/front-end/pull/25

File: okapi-ir/src/main/scala/org/opencypher/okapi/ir/impl/IRBuilder.scala [hunk 1]
```diff
@@ -194,33 +193,37 @@ object IRBuilder extends CompilationStage[ast.Statement, CypherStatement[Expr],
...
           refs <- {
             val onGraphs: List[QualifiedGraphName] = on.map(graph => QualifiedGraphName(graph.parts))
             val schemaForOnGraphUnion = onGraphs.foldLeft(Schema.empty) { case (agg, next) =>
               agg ++ context.schemaFor(next)
             }
 
-            // Computing single nodes/rels constructed by NEW (CREATE)
-            // TODO: Throw exception if both clone alias and original field name are used in NEW
-            val newPattern = newPatterns.foldLeft(Pattern.empty[Expr])(_ ++ _)
+            // Computing single nodes/rels constructed by CREATEA
```
Comment (s1ck) [inline okapi-ir/src/main/scala/org/opencypher/okapi/ir/impl/IRBuilder.scala]: typo

File: okapi-ir/src/test/scala/org/opencypher/okapi/ir/impl/IrBuilderTest.scala [hunk 2]
```diff
@@ -888,6 +889,195 @@ class IrBuilderTest extends IrTestSuite {
     }
   }
 
+  describe("SET in construct") {
+
+    // TODO: Ensure this semantic check happens in the frontend
```
Comment (s1ck) [inline okapi-ir/src/test/scala/org/opencypher/okapi/ir/impl/IrBuilderTest.scala]: Has this TODO been turned into a frontend card?

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/ConstructGraphPlanner.scala [hunk 3]
```diff
@@ -87,19 +87,26 @@ object ConstructGraphPlanner {
       case (op, (alias, original)) => op.retagVariable(alias, constructTagStrategy(original.cypherType.graph.get))
     }
 
-    // Construct NEW entities
-    val (newEntityTags, constructedEntitiesOp) = {
-      if (newEntities.isEmpty) {
+    // Construct CREATEd entities
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/ConstructGraphPlanner.scala]: typo

File: okapi-ir/src/test/scala/org/opencypher/okapi/ir/impl/IrBuilderTest.scala (hunk 2 above)
Comment (DarthMax) [reply okapi-ir/src/test/scala/org/opencypher/okapi/ir/impl/IrBuilderTest.scala]: yes

**Pull request merged**